pkg/commands/compute/compute_mocks_test.go

```diff
@@ -85,32 +80,3 @@ func getServiceDetailsWasm(i *fastly.GetServiceInput) (*fastly.ServiceDetail, er
 		Type: "wasm",
 	}, nil
 }
-
-type versionClient struct {
```

Comment:
As part of the consistency improvements and reduction in validation complexity, these test types are no longer used.